### ISSUE TITLE
Citra get's stuck when running Etrian Odyssey 2 Untold ( Europe ) dumped with ThreeSD or installed

### ISSUE THREAD

**Issue by Eduxki** [citra-emu/citra#5680]
Etrian Odyssey 2 Untold ( Europe ) doesn't start when the game is loaded as an installed game using the "virtual SD" that Citra creates, the log file becomes 51 mb size, this is the starting:

https://pastelink.net/2hqym

But when I decrypt the same dump as .cci and load it by load file option, game works perfectly.

Maybe this happens in other games too

**Comment by BreadFish64** [comment]
Does this happen if you install the game using a CIA dumped with GM9?

**Comment by Eduxki** [comment]
Yes, it happens dumping it with threesd and as .cia installed by Citra. 
When I launch it as an installed game, it doesn't get to start, fps are 0 and if I open the task manager, Citra app is using 1,7 Gb of memory and I have to stop it with task manager.

I verified the .cia with godmode9 and it passes all checks.

As I said, opening it by load file, it runs perfect, full speed.

My pc is i7 4770k and nvidia gtx 1080.

**Comment by Eduxki** [comment]
Video to see exactly what happens and ram usage increasing more and more:


https://user-images.githubusercontent.com/47615781/104109297-a9c50700-52cc-11eb-9444-e5206aeb29bf.mp4
